### ISSUE TITLE
[User flows and navigating] Link or button is not working as expected or did not navigate to the intended destination. (03.08.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#50797]
#### VFS team name
Sitewide Homepage

#### VFS product name
Redesigned homepage 

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Link or button is not working as expected or did not navigate to the intended destination.

### Issue Details
The external link in the banner labeled "Learn how an account helps you" has an empty anchor tag (missing the destination link) and doesn't navigate the user anywhere. 

### VA.gov Experience Standard
[Category Number 03, Issue Number 08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.3.1_A
WCAG SC 3.3.3_AA

### Platform Recommendation
Insert missing page link

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by jilladams** [comment]
Known issue ticketed as https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11771, and will be resolved prior to launch.
